ref-exchange/src/account_deposit.rs

```diff
@@ -238,6 +257,13 @@ impl Contract {
         self.accounts.insert(&account_id, &account.into());
     }
 
+    /// save token to owner account as lostfund
+    pub(crate) fn internal_lostfund(&mut self, token_id: &AccountId, amount: u128) {
+        let mut lostfund = self.internal_unwrap_or_default_account(&self.owner_id.clone());
+        lostfund.deposit(token_id, amount);
```

Comment:
If we switch `tokens` in all `accounts` into `UnorderedMap`, it would not only a main structure storage upgrade involved, but lazy account upgrade also needed be considered.  

How about this, We make this lostfound to be `UnorderedMap` and keep it as a direct member of contract structure.  In this way, the upgrade would be limited in the main structure scope and no existing user account would be touched.